src/app/dropdown/dropdown-angular-nested.demo.html

```diff
@@ -16,16 +16,16 @@
             <a href="javascript://" clrDropdownItem>Action 1</a>
             <a href="javascript://" class="disabled" clrDropdownItem>Disabled Action</a>
             <div class="dropdown-divider"></div>
-            <clr-dropdown [clrMenuPosition]="'right-bottom'">
+            <clr-dropdown>
                 <button clrDropdownToggle>Link 1</button>
```

Comment:
Are we renaming this to `clrDropdownTrigger`? Also do you think it would be better to rename all dropdown, tooltips, signposts triggers to just `clrTrigger`?